src/gpio.jl

```diff
@@ -0,0 +1,28 @@
+typealias gpio_context Ptr{Void}
+
+@enum(gpio_dir_t,
+  GPIO_OUT = 0,
+  GPIO_IN = 1,
+  GPIO_OUT_HIGH = 2,
+  GPIO_OUT_LOW = 3
+)
+
+function gpio_init(pin::Int)
```

Comment:
Yes, you are right. `Cint` is alias for `Int32`. I'll use the latter in the user-facing API.